src/main/java/me/totalfreedom/totalfreedommod/bridge/CoreProtectBridge.java

```diff
@@ -0,0 +1,241 @@
+package me.totalfreedom.totalfreedommod.bridge;
+
+import me.totalfreedom.totalfreedommod.FreedomService;
+import me.totalfreedom.totalfreedommod.TotalFreedomMod;
+import me.totalfreedom.totalfreedommod.util.FLog;
+import me.totalfreedom.totalfreedommod.config.ConfigEntry;
+import net.coreprotect.CoreProtect;
+import net.coreprotect.CoreProtectAPI;
+import org.bukkit.Bukkit;
+import org.bukkit.plugin.Plugin;
+import org.bukkit.World;
+import org.bukkit.scheduler.BukkitTask;
+import org.bukkit.scheduler.BukkitRunnable;
+import org.bukkit.plugin.PluginManager;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.sql.ResultSet;
+import java.io.File;
+import java.util.List;
+import java.util.Arrays;
+import me.totalfreedom.totalfreedommod.util.FUtil;
+
+public class CoreProtectBridge extends FreedomService
+{
+    private CoreProtectAPI coreProtectAPI = null;
+    
+    private final List<String> tables = Arrays.asList("co_sign", "co_session", "co_container", "co_block");
+    
+    private BukkitTask wiper;
+
+    public CoreProtectBridge(TotalFreedomMod plugin)
+    {
+        super(plugin);
+    }
+
+    @Override
+    protected void onStart()
+    {
+        final long interval = 10 * 20L;
+        final File databaseFile = getDatabase();
+
+        if (!ConfigEntry.COREPROTECT_AUTO_WIPING_ENABLED.getBoolean() || getCoreProtect() == null)
+        {
+            return;
+        }
+
+        wiper = new BukkitRunnable()
+        {
+            @Override
+            public void run()

```

Comment:
Why is all of this within the onStart section? This should probably be split out and then called here instead of being located here...?